### DISCUSSION TITLE
[Version 7.0] Paginator TreeTable column having SelectOneMenu does not render or loss the default/selected value from the managed bean on navigating to another paginator page tabs.

### DISCUSSION THREAD

**Discussion by Kapil-Gureja** [primefaces/community#2104] [PrimeFaces]
There appears to be some issue with the rendering on using selectonemenu inside paginator treetable. The column showing selectonemenu linked with the managed bean field value shows the correct value only on page1. On all other pages of the treetable, it does not show the correct value as already selected or as default value for the selectonemenu. Instead it shows as if there is no value being returned by the managed bean, selectonemenu with no value selected. Also on navigating back to page 1, again it does not show the selected value for the menu which was there initially on page 1. 

```xml
<p:treeTable value="#{createController.relationsModel.relationsRoot}"
			var="node" paginator="true" rows="10"
                        selectionMode="checkbox" id="resultRelations" 
			styleClass="ui-treetable-resultRelations hide-sel-chkbox"
			selection="#{createController.relationsModel.relationSelectedInRelationsTable}"
			resizableColumns="true">
<p:column style="width:10%">
				<h:outputText value="#{node.code}" rendered="true" />
</p:column>

<p:column
				rendered="#{createController.relationsModel.displayCategory}"
				style="width:10%">
			
				<h:outputText value="#{appCSMQBean.interpretCqtBaseCategory(node.category)}"
                    rendered="#{node.readOnlyCategory and not node.hideCategory}" style="word-wrap: break-word;" />
				<p:selectOneMenu id="soc2" value="#{node.category}" 
				        rendered="#{not node.readOnlyCategory}"
						autoWidth="false" style="width:75%" disabled="#{createController.readOnlyState}">
					<f:selectItem itemLabel="" itemValue="" noSelectionOption="true"/>
					<f:selectItems var="s" value="#{appCSMQBean.cqtBaseCategoriesWithInactiveSelected(node.category)}"
						itemValue="#{s.value}" itemLabel="#{s.value}" />
						<p:ajax event="change"
						listener="#{createController.notifyCategoryChange()}" update="@form:resultRelations"/>
				</p:selectOneMenu>
                
			</p:column>
```

The issue is only with the column having selectonemenu, not with the columns having outputText. Please advise if there is any bug with this? Thanks
<img width="918" alt="SS1" src="https://github.com/primefaces/community/assets/14810186/33c738b5-8a69-4302-8598-c70412c4c73c">
<img width="920" alt="SS2" src="https://github.com/primefaces/community/assets/14810186/bf01f226-520a-4ca8-9c93-0e5c40b7b62c">
<img width="940" alt="SS3" src="https://github.com/primefaces/community/assets/14810186/743b21da-4af8-4de4-9d44-378a96910787">

**Comment by melloware** [comment]
I am pretty sure this is fixed in future PF versions.  Can you try 14.0.0?

Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.

**Comment by melloware** [comment]
Closing as no response from OP.
